contracts/FlashMintLib.sol

```diff
@@ -0,0 +1,154 @@
+pragma solidity 0.6.12; 
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/math/Math.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./Interfaces/Compound/CErc20I.sol";
+import "./Interfaces/Compound/ComptrollerI.sol";
+import "./Interfaces/DAI/IERC3156FlashLender.sol";
+import "./Interfaces/DAI/IERC3156FlashBorrower.sol";
+
+interface IWETH is IERC20 {
+	function deposit() payable external;
+	function withdraw(uint256) external;
+}
+
+interface IUniswapAnchoredView {
+	function price(string memory) external returns (uint);
+}
+
+interface IERC20Extended is IERC20 {
+	function decimals() external view returns (uint8);
+
+	function name() external view returns (string memory);
+
+	function symbol() external view returns (string memory);
+}
+
+library FlashMintLib {
+	using SafeMath for uint256;
+	event Leverage(uint256 amountRequested, uint256 amountGiven, bool deficit, address flashLoan);
+
+	uint256 private constant PRICE_DECIMALS = 1e6;
+	uint256 private constant DAI_DECIMALS = 1e18;
+	uint256 private constant COLLAT_RATIO_DAI = 0.74 ether;
```

Comment:
not convinced about hardcoding this. it could and probably will change in a gov proposal at some point